pkg/hoist/hoist_launchable.go

```diff
@@ -33,6 +34,10 @@ type EntryPoints struct {
 	Implicit bool
 }
 
+// IncludePodIDArg will include a --podID argument to p2-exec.
+// Delaying the use of this argument from the rollout of the binary that understands it allows for n-1 version compatibility
+var IncludePodIDArg = param.Bool("include_pod_id_arg", false)
```

Comment:
I'd like to re-use the same config value (nested_cgroups), but I cannot import the pods package here.

This isn't a huge deal as this param should be short lived.